### ISSUE TITLE
Add a toggle switch to temporarily disable updating when changes are pushed to database

### ISSUE THREAD

**Issue by honoki** [honoki/bbrf-dashboard#1]
At the moment the dashboard automatically refreshes the data tables when updates to the database are detected, but this can get annoying when trying to work with the dashboard while pushing results to the server.

A toggle to temporarily disable auto-updates, optionally with a manual refresh button, would be nice to avoid this scenario.

**Comment by pdelteil** [comment]
Yes, sometimes the dashboard behaves oddly, when I select an item from the program's combobox it would jump to other item on the list. I'm not sure if the root cause is the updating process.
